### ISSUE TITLE
Client not usable by default?

### ISSUE THREAD

**Issue by craigbruce** [Binstar/binstar#110]
I've read the docs, but I must be missing something because I pip install binstar (v0.10.0) but I can't do anything, `binstar --help`, `binstar upload` or `binstar login` all yield this same error:

```
(binstar)craig@daedalus:~ $ binstar login
Traceback (most recent call last):
  File "/Users/craig/.virtualenvs/binstar/bin/binstar", line 9, in <module>
    load_entry_point('binstar==0.10.0', 'console_scripts', 'binstar')()
  File "/Users/craig/.virtualenvs/binstar/lib/python2.7/site-packages/binstar_client/scripts/cli.py", line 94, in main
    description=__doc__, version=version)
  File "/Users/craig/.virtualenvs/binstar/lib/python2.7/site-packages/binstar_client/scripts/cli.py", line 60, in binstar_main
    add_subparser_modules(parser, sub_command_module)
  File "/Users/craig/.virtualenvs/binstar/lib/python2.7/site-packages/clyent/__init__.py", line 115, in add_subparser_modules
    for command_module in get_sub_commands(module):
  File "/Users/craig/.virtualenvs/binstar/lib/python2.7/site-packages/clyent/__init__.py", line 105, in get_sub_commands
    this_module = __import__(module.__package__ or module.__name__, fromlist=names)
  File "/Users/craig/.virtualenvs/binstar/lib/python2.7/site-packages/binstar_client/commands/remove.py", line 12, in <module>
    from binstar_client import NotFound
ImportError: cannot import name NotFound
```
Am I missing something obvious?

**Comment by srossross** [comment]
@craigbruce sorry for the inconvenience, this error has been fixed in version `0.10.1`

**Comment by craigbruce** [comment]
Excellent! That is working now, thanks for the quick resolution.
